### ISSUE TITLE
Proposal: Begin using Jekyll

### ISSUE THREAD

**Issue by orawalters** [KeepOurNetFree/keepournetfree.github.io#2]
The CMS that GH-Pages already autobuilds. Should make development easier without requiring anymore resources.

**Comment by jacobherrington** [comment]
I agree. If you're planning to include a blog or do anything other than host this static site, it would be smart to use Jekyll.
